frontend/common/dispatcher/app-actions.js

```diff
@@ -21,9 +21,10 @@ const AppActions = Object.assign({}, require('./base/_app-actions'), {
       identity,
     })
   },
-  confirmTwoFactor(pin, onError) {
+  confirmTwoFactor(pin, onError, isLoginPage) {
```

Comment:
Im a bit confused as to why any frontend code was changed, we already have 2fa, shouldn't this just use what was there?